CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required (VERSION 3.1)
 project (CleverLeaf CXX Fortran)
+ENABLE_LANGUAGE(C)
```

Comment:
Hey @twang15 - instead of using `enable_language`, can you just add `C` to the project command:

`project (CleverLeaf CXX C Fortran)`

Reply:
Interesting, I actually tried that before, but it did not succeed.
Now I tried it again, it works now.